.github/CODEOWNERS

```diff
@@ -4,6 +4,7 @@
 /**/OWNERS @markdroth @nicolasnoble @a11r
 /bazel/** @nicolasnoble @dgquintas @a11r @vjpai
 /cmake/** @jtattermusch @nicolasnoble @matt-kwong
+/cmake/generated/** @grpc/grpc-team
```

Comment:
Maybe we can do something like changing line 6 to `/cmake/[A-Za-z_.]+$ @jtattermusch @nicolasnoble @matt-kwong` if what we're trying to do is let anyone approve changes to generated files in `/cmake`, but we want owners approval for everything else. This assumes we don't add any additional subdirectories to `/cmake`, so it might cause issues in the future.